schema_utils.py

```diff
@@ -616,8 +616,7 @@ def is_supported_audio_language_code(obj):
         return utils.is_supported_audio_language_code(obj)
 
     @staticmethod
-    def is_valid_audio_language_code(obj):
-        # type: (Text) -> bool
+    def is_valid_audio_language_code(obj: Text) -> bool:
```

Comment:
```suggestion
    def is_valid_audio_language_code(obj: str) -> bool:
```

Please use `str` instead of `Text`

Reply:
ditto elsewhere

Reply:
Updated.